src/panels/lovelace/cards/hui-conditional-card.ts

```diff
@@ -32,7 +32,9 @@ class HuiConditionalCard extends HTMLElement implements LovelaceCard {
     this.card = createCardElement(config.card);
     if (this.card) {
       this.appendChild(this.card);
-      this.card.hass = this.hass;
+      if (this.hass) {
```

Comment:
this is weird. There is no `this.hass` to get, I only see a setter, no getter.  Neither is the setter storing the value ot `this._hass`.